### PR TITLE
`lang`: Add Linux pre-built Python interpreter

### PR DESCRIPTION
**Pull request by razvand**
Add a Linux pre-built Python interpreter. Add pre-built ELF, pre-etracted dynamic libraries, Python libraries archive (in `usr.tar.gz`).  Document steps to run in `README.md`.

### CODE REVIEW

File: lang/python/README.md [hunk 1]
```diff
@@ -1 +1,99 @@
...
+>>>
+```
+
+You can exit the console with the `exit()` command or the `Ctrl+d` keyboard shortcut.
+
+## Run HTTP Server on Linux
+
+Run the HTTP server implemented in `http_server.py` by running:
+
+```console
```
Comment (StefanJum) [inline lang/python/README.md:54]:
```suggestion
```console
./python http_server.py
```

**Review by StefanJum — CHANGES_REQUESTED** [review]
I get some weird errors when trying to run the server on Unikraft (it works fine if I run it natively on linux):
```
>>>   File "//http_server.py", line 1
    try:
IndentationError: unexpected indent
>>>   File "//http_server.py", line 1
    webServer.serve_forever()
IndentationError: unexpected indent
>>>   File "//http_server.py", line 1
    except KeyboardInterrupt:
IndentationError: unexpected indent
>>>   File "//http_server.py", line 1
    pass
IndentationError: unexpected indent
>>> >>>   File "//http_server.py", line 1
    webServer.server_close()
IndentationError: unexpected indent
>>>   File "//http_server.py", line 1
    print("Server stopped.")
IndentationError: unexpected indent
```

Also, I think the `custom-build` directory should be named `python-custom-build` or should be places inside `lang/python/`.

**Comment by razvand** [comment]
@StefanJum , I'm checking the issue now. It may be because of an outdated app-elfloader build.

**Review by StefanJum — APPROVED** [review]
Reviewed-by: Stefan Jumarea <stefanjumarea02@gmail.com>
Approved-by: Stefan Jumarea <stefanjumarea02@gmail.com>

**Comment by StefanJum** [comment]
Will leave the http server issue as it is, since it is not related to this pr.

**Pull request merged**